docs/index.md

```diff
@@ -1,91 +1,116 @@
-# Coyote
-
-Can you find the deadlock bug in this BoundedBuffer class?
-
-```c#
-public class BoundedBuffer
-{
-    private readonly object SyncObject = new object();
-    private readonly object[] Buffer;
-    private int PutAt;
-    private int TakeAt;
-    private int Occupied;
-
-    public BoundedBuffer(int bufferSize)
-    {
-        this.PulseAll = pulseAll;
-        this.Buffer = new object[bufferSize];
+# Welcome to Coyote
+
+Coyote is a set of .NET libraries and tools designed to help ensure that your code is free of bugs.
+Too often developers are drowning in the complexity of their own code and many hours are wasted
+trying to track down impossible-to-find bugs, especially when dealing with _concurrent_ code or
+various other sources of _non-determinism_ (like message ordering, failures, timeouts and so on).
+
+Coyote provides programming models to express concurrent systems. These programming models offer
+convenient ways to program at a high-level of abstraction. As mentioned below, Coyote currently
+supports two programming models: a familiar tasks-based programming model (currently in-preview) as
```

Comment:
Ah, Chris could you remove this bit? We forgot to remove this months ago! `(currently in-preview)`